src/index.js

```diff
@@ -0,0 +1,96 @@
+// change the scene depending on the sky
+// make a reset city name button
+// add a "landscape" element
+
+const state = {
+  clickCount: 50,
+};
+
+const colors = (clicks, text) => {
+  if (clicks < 50) {
+    text.style.color = 'blue';
+  } else if (clicks < 59) {
+    text.style.color = 'green';
+  } else if (clicks < 69) {
+    text.style.color = 'pink';
+  } else if (clicks < 79) {
+    text.style.color = 'orange';
+  } else {
+    text.style.color = 'red';
+  }
+};
+
+const landscape = (clicks, container) => {
+  if (clicks < 50) {
+    container.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (clicks < 59) {
+    container.textContent = '🍁🍃🍂🍁🍃🍂🍁🍃🍂🍁🍃🦙';
+  } else if (clicks < 69) {
+    container.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (clicks < 79) {
+    container.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else {
+    container.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  }
+};
+const increaseClickCount = () => {
+  const displayTemp = document.getElementById('temp');
+  const landscapeDisplay = document.getElementById('landscape-scene');
+  state.clickCount += 1;
+  displayTemp.textContent = state.clickCount;
+  console.log('up');
+  colors(state.clickCount, displayTemp);
+  landscape(state.clickCount, landscapeDisplay);
+};
```

Comment:
Notice how similar this function is to `decreaseClickCount`. Try to refactor these methods so that the shared parts are in a separate helper function that can be called by either the increase or decrease method.